aten/src/ATen/native/cuda/Embedding.cu

```diff
@@ -332,10 +334,11 @@ Tensor & embedding_renorm_cuda_(Tensor & self, const Tensor & indices,
   dim3 block(128);
   int dim = self.stride(0);
 
-  DISPATCH_ALL_FLOATING_TYPES(self.type(), "embedding_backward", [&]() {
-    using accscalar_t = cuda::acc_type<scalar_t>;
+  AT_DISPATCH_FLOATING_TYPES_AND_HALF(self.type(), "embedding_backward", [&] {
+    using cuda_scalar_t = to_cuda_type<scalar_t>::type;
```

Comment:
maybe copy the pattern for `cuda::acc_type` (e.g. `cuda::type<scalar_t>`)? I didn't realize we had `acc_type` now but it is serving a similar purpose.